python/tvm/autotvm/record.py

```diff
@@ -152,7 +152,6 @@ def decode(row, protocol="json"):
         tgt, task_name, task_args, task_kwargs = row["input"]
         tgt = str(tgt)
         if "-target" in tgt:
-            logger.warning('"-target" is deprecated, use "-mtriple" instead.')
```

Comment:
do we want to remove this line?

Reply:
It was noisy so I deleted it locally haha! I'll put it back.

Reply:
I comment this out for all of my development, I think I've accidentally pushed it to 4 PRs now.
We need to fix the issues that raise this warning